package.json

```diff
@@ -39,11 +40,17 @@
     "webpack-dev-server": "^4.7.4"
   },
   "peerDependencies": {
-    "kaltura-player-js": "https://github.com/kaltura/kaltura-player-js.git#master"
+    "kaltura-player-js": "https://github.com/kaltura/kaltura-player-js.git#3.8.0"
   },
   "publishConfig": {
     "access": "public"
   },
+  "kaltura": {
+    "name": "related",
+    "dependencies": {
+      "playkit-ui-managers": "1.3.0"
```

Comment:
@SivanA-Kaltura currently it is 1.3.1
I think it may change to 1.3.2 after 1/1 release

Reply:
then we'll need to update the version in every dependent project

Reply:
you're right, i added the dependencies before the release